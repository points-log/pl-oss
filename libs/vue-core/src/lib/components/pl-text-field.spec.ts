import { shallowMount } from '@vue/test-utils';
import Vue from 'vue';
import Vuetify from 'vuetify';
import PLTextField from './pl-text-field.vue';

Vue.use(Vuetify);

describe('PLTextField', () => {
  it('should match the snapshot', () => {
    const wrapper = shallowMount(PLTextField);
    expect(wrapper.element).toMatchSnapshot();
  });
});
