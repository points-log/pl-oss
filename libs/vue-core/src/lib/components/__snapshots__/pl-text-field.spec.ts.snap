// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PLTextField should match the snapshot 1`] = `
<v-text-field-stub
  backgroundcolor=""
  clearicon="$clear"
  errorcount="1"
  errormessages=""
  hidedetails="auto"
  loaderheight="2"
  messages=""
  outlined="true"
  rules=""
  successmessages=""
  type="text"
/>
`;
